php/libraries/NDB_Form.class.inc

```diff
@@ -176,7 +176,7 @@ class NDB_Form extends NDB_Page
      */
     function _resetForm()
     {
-        $this->_setupPage($name, $page, $identifier, null, 'test_form');
+        $this->_setupPage(null, null, null, null, 'test_form');
```

Comment:
I don't know if resetForm is used anywhere, but since the 3 variables aren't defined, this is what it was doing anyways.